.github/workflows/CI.yml

```diff
@@ -0,0 +1,71 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  style:
+    name: Check Style
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Install Rust
```

Comment:
what about using a rust-toolchain file?

Reply:
H2 has some checks that require nightly...
https://github.com/hyperium/h2/blob/3cfcab016e17f36884ee77ff85947402535468ee/.travis.yml#L47-L51